typing/index.d.ts

```diff
@@ -0,0 +1,47 @@
+export declare type TypeDefinition = object;
```

Comment:
@AFaugeras someone does some typing for a lib that I wrote, and declared a module on top of all declarations : https://github.com/fabienjuif/use-bus/blob/master/index.d.ts

I think this is a must have so we don't have type collisions.
What is your though?

Reply:
@fabienjuif Sorry, didn't see you comment earlier 😕

I'm going to give it a try abd update my PRs subsequently 😉